keyboards/kudox_game/keymaps/default/keymap.c

```diff
@@ -35,10 +44,33 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
   //┌────────┼────────┼────────┼────────┼────────┼────────┼────────┐
      _______ ,_______ ,KC_LEFT ,KC_DOWN ,KC_RIGHT,KC_LBRC ,KC_RBRC ,
   //├────────┼────────┼────────┼────────┼────────┼────────┼────────┐
-     _______ ,KC_BSPC                   ,KC_CIRC ,KC_LPRN ,KC_RPRN ,
+     _______ ,KC_BSPC                   ,KC_DEL  ,KC_LPRN ,KC_RPRN ,
   //├────────┼────────┘                 └────────┴────────┼────────┼────────┐
      _______                                              ,_______ ,_______
   //└────────┘                                            └────────┴────────┘
+  ),
+
+  [_LIGHT] = LAYOUT(
+  //┌────────┬────────┬────────┬────────┬────────┬────────┐
+     _______ ,RGB_HUI ,RGB_HUD ,RGB_SAI ,RGB_SAD ,RGB_VAI ,
+  //└────────┼────────┼────────┼────────┼────────┼────────┤
+              RGB_M_P ,RGB_M_SW,RGB_M_X ,RGB_M_B ,RGB_VAD ,
+  //┌────────┼────────┼────────┼────────┼────────┼────────┼────────┐
+     _______ ,_______ ,RGB_M_SN,RGB_M_G ,RGB_M_R ,RGB_M_K ,RGB_M_T ,
+  //├────────┼────────┼────────┼────────┼────────┼────────┼────────┐
+     _______ ,RGB_M_T                   ,XXXXXXX ,RGB_VAI ,RGB_VAD ,
+  //├────────┼────────┘                 └────────┴────────┼────────┼────────┐
+     _______                                              ,RGB_MOD ,RGB_TOG
+  //└────────┘                                            └────────┴────────┘
   )
 
 };
+
+// Runs just one time when the keyboard initializes.
+#ifdef RGBLIGHT_ENABLE
+void matrix_init_user(void) {
+    rgblight_init();
+    rgblight_setrgb(0x00,  0xFF, 0xFF);
+    rgblight_mode_noeeprom(RGBLIGHT_MODE_RAINBOW_SWIRL);
+};
```

Comment:
This isn't the right way to do this.  Rather, this should be: 
```suggestion
void keyboard_post_init_user(void) {
    rgblight_mode_noeeprom(RGBLIGHT_MODE_RAINBOW_SWIRL);
}
```

This will run after the rgb light code has initialized.

Reply:
@drashna Thank you for reviewing!
I modified your suggested points.